### ISSUE TITLE
Crit Value System: Flawed

### ISSUE THREAD

**Issue by NoelWintory** [Daripher/Passive-Skill-Tree#197]
I've been conducting experiments into the balance of this mod, and I noticed that the crit values do not affect Minecraft's crit system. A crafted food buff from the crafted skill tree does not add 20% crit damage to the normal 150% Minecraft crit damage; instead, it adds 20% for a minute to a 300% crit damage value that becomes active whenever you have a skill unlocked that boosts crit chance. 

There is a reason Minecraft's developers did not make crit damage do 300% of a normal strike and (2(x(300+a))/3 for critical strikes, were x is the base damage of a weapon and a is the crit damage bonus from skills. If nothing else, this completely and utterly breaks the mod with apotheosis. I don't know how you thought this was a good idea to implement, I'll be honest, but regardless of the story behind it, please consider either applying the bonuses and effects to Minecraft's crit and crit damage systems, or at least modifying the system that seems to be provided by Passive Skill Tree to reflect Minecraft's crit system values. 

If nothing else, please incorporate a config that allows players to modify the crit system accordingly. 

Overall though, love the work and I thank you for your consideration and time. The current mod-pack we are using has been provided, should the crit system not be an intentional addition.

[God_Race_1.20.1_Final (April26).zip](https://github.com/Daripher/Passive-Skill-Tree/files/15137436/God_Race_1.20.1_Final.April26.zip)

**Comment by Daripher** [comment]
300% crit damage multiplier was never intended, this is a bug.

**Comment by NoelWintory** [comment]
I had a feeling that might be the case. There is always a story to this
kind of thing.

Looking at your code, I think I want to get involved in coding and try my
hand at solving the problem myself unofficially—for temporary use on my own
server until you're able to resolve it on your end. I tried looking through
to see if I could find the file with the modifier (hoping it was an easy
fix) but couldn't find it. The fix isn't so simple in this case, is it? I'm
not 100% sure what a float is, save for that it allows for decimal values
in javascript, but what I was able to find so far in the script is mention of "float." Is that where the problem is? 

I'm genuinely curious and want to understand, but don't let take up too much of your time. I'll figure it out eventually regardless.
